test/ReactWrapper-spec.jsx

```diff
@@ -2073,17 +2073,37 @@ describeWithDOM('mount', () => {
   });
 
   describe('.hasClass(className)', () => {
-    it('should return whether or not node has a certain class', () => {
-      const wrapper = mount(
-        <div className="foo bar baz some-long-string FoOo" />,
-      );
+    context('When using a DOM component', () => {
```

Comment:
`context` and `describe` are identical - either is fine, but i have a light preference for `describe`.